Transmit launch_presentation_locale parameter

haikuginger opened the pull request:
This is related to [LOC-78](https://openedx.atlassian.net/browse/LOC-78).

When using the edX LMS as a frontend for other LTI modules, this change enables us to transmit the user's preferred language setting (if present) as part of the POST request to set the iframe up.

If the user doesn't have a language set on their profile, or if they don't have an edX LMS profile set up, we won't transmit locale preferences, and the behavior will remain the same.

This PR is covered by the OpenCraft contributor agreement - thanks @antoviaque!

edx-webhook commented:
Thanks for the pull request, @haikuginger! I've created [OSPR-1272](https://openedx.atlassian.net/browse/OSPR-1272) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx/xblock-lti-consumer/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

nedbat commented:
@haikuginger thanks, I've added you as an OpenCraft prospect in our contributor database.

@douglashall is this still in your wheelhouse?

douglashall commented on lti_consumer/tests/unit/test_lti.py:
```diff
@@ -181,14 +182,19 @@ def test_get_signed_lti_parameters(self):
         self.assertEqual(self.lti_consumer.get_signed_lti_parameters(), expected_lti_parameters)
 
         # Test that `lis_person_sourcedid` and `lis_person_contact_email_primary` are not included
         # in the returned LTI parameters when a user cannot be found
```
Please update this comment.

douglashall commented:
👍 After comment addressed. Thanks for your contribution!

haikuginger commented:
Updated the comment in `test_lti.py` as well as in `lti.py`.

douglashall commented:
@haikuginger Sorry, one more thing. Can you bump the setup.py version and create a corresponding PR in edx-platform bumping the version of this package [here](https://github.com/edx/edx-platform/blob/master/requirements/edx/github.txt#L92)?

douglashall commented:
@haikuginger Thanks for updating the version. Two last things, I promise. Can you squash commits and create the v1.0.7 tag?

haikuginger commented:
I've squashed the commits down and added a tag to my fork, but from [what I can find](http://stackoverflow.com/questions/12278660/adding-tags-to-a-pull-request), tags don't appear to come across in the PR.

douglashall commented:
@haikuginger Ah, good point, forgot we are working with a fork here. I can take care of tagging and creating the edx-platform PR to update the requirement. Thanks.

haikuginger commented:
Oops, already opened the PR over there - edx/edx-platform#12609.

douglashall commented:
@haikuginger No worries. I will merge this, create the tag, and then merge the edx-platform PR.

haikuginger commented:
Thanks!
